frontend/src/components/ccofApplication/group/LicenseUpload.vue

```diff
@@ -169,7 +176,7 @@ export default {
       await this.save();
     },
     async save() {
-      this.processing = true;
+      this.isProcessing = true;
```

Comment:
the tag: loading is checking for "processing" but on save you're setting isProcessing.  can you confirm?